translations/base-fr.yaml

```
@@ -49,29 +49,29 @@ steamPage:
             [*] Balises infinies
             [*] Parties infinies
             [*] Plus d'options
-            [*] Prochainement: Câbles et énergie ! Prévu pour (environ) fin Juillet 2020.
-            [*] Prochainement: Plus de niveaux
+            [*] Prochainement : Câbles et énergie ! Prévu pour (environ) fin juillet 2020.
+            [*] Prochainement : Plus de niveaux
             [*] Aidez moi à continuer de développer shapez.io ❤️
         [/list]
 
-        [b]Mises à jour futures[/b]
+        [b]Mises à jour à venir[/b]
 
-        Je fais souvent des mises à jour et essaye d'en sortir une par semaine!
+        Je fais souvent des mises à jour et essaye d'en sortir une par semaine !
```

```suggestion
        Je fais souvent des mises à jour et j'essaye d'en sortir une par semaine !
```
You can't skip the subject here, you must put it a second time otherwise it's not french :)

---

Done, but with Naheulf's fancy typesetter's apostrophe to be consistent.

(Also, why say it's "not French"? There are many examples of native French speakers using such constructions. Can't you just argue it's better style?)